style.css

```diff
@@ -0,0 +1,54 @@
+/* @media only screen and (max-width: 600px) {
+    .main-nav {
+        font-size: 0.5em;
+        padding: 0;
+    }
+} */
```

Comment:
Similarly, like in HTML code, it would look cleaner if you can remove unused code like this.

Reply:
- [ ] Good work 😄